source/api/presentation/3.0/index.md

```diff
@@ -682,6 +682,10 @@ The value of this property _MUST_ be an array of JSON objects, each of which _MU
  * Any resource type _MAY_ have the `homepage` property.<br/>
    Clients _SHOULD_ render `homepage` on a Collection, Manifest or Canvas, and _MAY_ render `homepage` on other resource types.
 
+__Model Alignment__<br/>
+Please note that the [Web Annotation Data Model][org-w3c-webanno] has less stringent requirements about the model to be used for the `homepage` property. The IIIF requirements are compatible, but the home page of an Agent found might have only a URI, or might be a JSON object with other properties.
+{: .note}
```

Comment:
Small thing, but I wonder if it might be nicer to say, rather than

> the Web Annotation Data Model has less stringent requirements

That IIIF further refines the WADM?

Reply:
:+1:

Reply:
Also added forward ref to 4.7